Doc/whatsnew/3.14.rst

```diff
@@ -171,6 +171,14 @@ See the :ref:`JSON command-line interface <json-commandline>` documentation.
 (Contributed by Trey Hunner in :gh:`122873`.)
 
 
+multiprocessing
+---------------
+
+* Add the :attr:`~multiprocessing.Process.closed` property to
+  :class:`multiprocessing.Process` objects to determine whether
+  they have been closed or not.
+
```

Comment:
You should re-add the "Contributed by [...]" part that you had. It was correct before.